### PR TITLE
Update JeongHyeon Lee's info and references to NLP

### PR DESCRIPTION
**Pull request by coding-Benny**
The following contents have been updated according to the request for the 1st week assignment.

1. I have added my name and github link to `aboutme.md`.
2. I have added useful materials for studying git in `2021-07-22-course1-git-and-github.md`.
3. I have added a reference for text clustering and a description of text summarization to `2021-07-23-course2-about-nlp.md`.
4. I have added links to open source NLP tools for Korean and multilingual in `2021-08-05-course3-korean-nlp-tools.md`.

### CODE REVIEW

File: _posts/2021-07-22-course1-git-and-github.md [hunk 1]
```diff
@@ -22,14 +22,18 @@ Git, Github을 간단히 배울 수 있는 링크입니다.
 - [Github 사용 설명서, bnitech](https://bnitech.tistory.com/9)
 - [official, helloworld](https://guides.github.com/activities/hello-world/)
 - [official, lab](https://lab.github.com/)
+- [팀 개발을 위한 Git, GitHub 시작하기](http://www.yes24.com/Product/Goods/86031178)
```
Comment (hexists) [inline _posts/2021-07-22-course1-git-and-github.md]: @coding-Benny 
좋은 링크 남겨주셔서 감사합니다.
다른 링크와 달리 책 구매 링크임을 표시하면 어떨까요?
cc. @bab2min

Comment (coding-Benny) [reply _posts/2021-07-22-course1-git-and-github.md]: @hexists

말씀해 주신 대로 책 구매 링크임을 표시하도록 수정하여 올리도록 하겠습니다!
피드백 감사합니다 :)

**Comment by hexists** [comment]
@coding-Benny 
ebook 링크 추가해주셔서 감사합니다. 그리고 conflict 해결도 해주셨네요^^
이대로 merge 하겠습니다.
cc. @bab2min

**Comment by coding-Benny** [comment]
@hexists 
네 확인했습니다! 감사합니다😊

**Pull request merged**